<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ArtistInfoWidget</class>
 <widget class="QWidget" name="ArtistInfoWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>965</width>
    <height>844</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_3">
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <property name="leftMargin">
      <number>16</number>
     </property>
     <property name="topMargin">
      <number>12</number>
     </property>
     <property name="rightMargin">
      <number>16</number>
     </property>
     <property name="bottomMargin">
      <number>12</number>
     </property>
     <item>
      <layout class="QVBoxLayout" name="verticalLayout">
       <item>
        <widget class="QLabel" name="label">
         <property name="font">
          <font>
           <pointsize>16</pointsize>
           <weight>75</weight>
           <bold>true</bold>
          </font>
         </property>
         <property name="text">
          <string>Top Hits</string>
         </property>
         <property name="margin">
          <number>0</number>
         </property>
        </widget>
       </item>
       <item>
        <widget class="PlaylistView" name="topHits">
         <property name="minimumSize">
          <size>
           <width>0</width>
           <height>300</height>
          </size>
         </property>
         <property name="headerHidden">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="label_2">
         <property name="font">
          <font>
           <family>Arial</family>
           <pointsize>16</pointsize>
           <weight>75</weight>
           <bold>true</bold>
          </font>
         </property>
         <property name="text">
          <string>Albums</string>
         </property>
         <property name="margin">
          <number>0</number>
         </property>
        </widget>
       </item>
       <item>
        <widget class="GridView" name="albums">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="minimumSize">
          <size>
           <width>0</width>
           <height>190</height>
          </size>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="label_3">
         <property name="font">
          <font>
           <family>Arial</family>
           <pointsize>16</pointsize>
           <weight>75</weight>
           <bold>true</bold>
           <kerning>true</kerning>
          </font>
         </property>
         <property name="text">
          <string>Related Artists</string>
         </property>
         <property name="margin">
          <number>0</number>
         </property>
        </widget>
       </item>
       <item>
        <widget class="GridView" name="relatedArtists">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="minimumSize">
          <size>
           <width>0</width>
           <height>190</height>
          </size>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QVBoxLayout" name="verticalLayout_2">
       <property name="leftMargin">
        <number>16</number>
       </property>
       <item>
        <widget class="QLabel" name="cover">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="minimumSize">
          <size>
           <width>220</width>
           <height>220</height>
          </size>
         </property>
         <property name="text">
          <string>Cover</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignCenter</set>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QTextBrowser" name="biography">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Minimum" vsizetype="MinimumExpanding">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="minimumSize">
          <size>
           <width>0</width>
           <height>240</height>
          </size>
         </property>
         <property name="verticalScrollBarPolicy">
          <enum>Qt::ScrollBarAlwaysOff</enum>
         </property>
         <property name="horizontalScrollBarPolicy">
          <enum>Qt::ScrollBarAlwaysOff</enum>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>PlaylistView</class>
   <extends>QTreeView</extends>
   <header>playlist/PlaylistView.h</header>
  </customwidget>
  <customwidget>
   <class>GridView</class>
   <extends>QListView</extends>
   <header location="global">GridView.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
